### PR TITLE
Change to more up to date link for Ethnicity harmonised standard

### PR DESCRIPTION
**Pull request by edwardhorsford**
GSS appear to have moved to a new site - the link was pointing at the old (and out of date) archived standard.

See #2284 for more relevant conversation.

### CODE REVIEW

**Comment by 36degrees** [comment]
@edwardhorsford if this was merged would it resolve #2284?

**Comment by edwardhorsford** [comment]
> @edwardhorsford if this was merged would it resolve #2284?

No it wouldn't. The pattern is still using the harmonised standard from 2011, where GSS now recommends teams may want to use the Census questions instead.

However, in linking to the correct version of the standard, teams may at least see the published notes about the standard being revised and how it relates to the 2021 Census.

**Comment by domoscargin** [comment]
Thanks, @edwardhorsford!

Should this link be to the overview at https://analysisfunction.civilservice.gov.uk/government-statistical-service-and-statistician-group/gss-support/gss-harmonisation-support/harmonised-standards-and-guidance/?

Since this pattern covers:

>- age
>- disability
>- ethnic group
>- marital or partnership status
>- religion
>- sex and gender identity
>- sexual orientation

File: src/patterns/equality-information/index.md.njk
Comment (calvin-lau-sig7) [inline src/patterns/equality-information/index.md.njk:19]:
```suggestion
These patterns are based on the [harmonised standards developed by the Government Statistical Service](https://analysisfunction.civilservice.gov.uk/government-statistical-service-and-statistician-group/gss-support/gss-harmonisation-support/harmonised-standards-and-guidance/).
```

**Comment by edwardhorsford** [comment]
Thanks @domoscargin and @calvin-lau-sig7 - yeah I had the wrong link. Now fixed with @calvin-lau-sig7's suggestion.

**Review by calvin-lau-sig7 — APPROVED** [review]

**Review by domoscargin — APPROVED** [review]
Looks good to me!

Just need another review from the team before merging.

Edit: No we don't! Thanks @calvin-lau-sig7

**Pull request merged**
